srcCpp03/RTIDDSImpl.cxx

```diff
@@ -555,7 +580,7 @@ class RTIPublisher: public RTIPublisherBase<T> {
             this->data.key()[c] = (unsigned char) (key >> c * 8);
         }
         if (!isCftWildCardKey) {
-            this->_writer.write(this->data, this->_instance_handles[key]);
+            this->_writer.write(this->data, this->_instance_handles.back());
```

Comment:
the way to select the instance handle calls for a comment. This code is not self-descriptive.